README.md

```diff
@@ -456,7 +456,10 @@ ANSWERS.addComponent('SearchBar', {
     enabled: false,
     // Optional, the message in the alert. Defaults to the below
     message: "We are unable to determine your location"
-  }
+  },
+  // Optional, a callback invoked when a search is submitted via this component. The users search
+  //           is passed in as a string
```

Comment:
Minor nit: "Optional" and "is passed in" should be the same level of indentation.